.github/workflows/beta.yml

```diff
@@ -32,13 +32,14 @@ jobs:
           override: true
           components: clippy
 
-      - name: Run clippy
+      - name: Run Clippy
         uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: --all-features --workspace --tests --examples -- -D clippy::all
```

Comment:
afaik dropping `clippy::all` will make it a lot more lenient, is that intentional?

Reply:
According to docs, that's the default now, so `-D warnings` includes `clippy::all` and rust's own lints. When we run rust-analyzer locally, we also no longer need to specify `clippy::all`.

@olksdr and I were wondering too and tested this locally.